media-gfx/rawtherapee/rawtherapee-5.8.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_P=${P/_rc/-rc}
+inherit cmake flag-o-matic toolchain-funcs xdg-utils
+
+DESCRIPTION="A powerful cross-platform raw image processing program"
+HOMEPAGE="https://www.rawtherapee.com/"
+SRC_URI="https://rawtherapee.com/shared/source/${MY_P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="openmp tcmalloc"
+
+RDEPEND="
+	dev-libs/expat
+	dev-libs/libsigc++:2
+	media-libs/lcms:2
+	media-libs/lensfun
+	media-libs/libcanberra[gtk3]
+	media-libs/libiptcdata
```

Comment:
media-libs/libiptcdata is a py2-only package. Is there a way to make it optional?

Reply:
I'd say it's difficult to make it optional (deeply embedded in rtexif.{cc,h}) and also still relevant for an imaging software to speak IPTC (used by Adobe extensively)
@heckflosse, @Beep6581 do you have anything to say?

Reply:
It is not optional. Ideally we'd like to drop it entirely as libiptcdata is dead, but for the time being it's required.